src/Windows/Avalonia.Win32/Interop/Automation/IRawElementProviderSimple2.cs

```diff
@@ -1,11 +1,52 @@
-﻿using System.Runtime.InteropServices;
+﻿using System;
+using System.Runtime.InteropServices;
+using Avalonia.Win32.Automation;
 
 namespace Avalonia.Win32.Interop.Automation
 {
     [ComVisible(true)]
     [Guid("a0a839a9-8da1-4a82-806a-8e0d44e79f56")]
-    public interface IRawElementProviderSimple2
+    public interface IRawElementProviderSimple2 : IRawElementProviderSimple
```

Comment:
If this is supposed to match the `IRawElementProviderSimple2`, then the offsets here are incorrect as it should have the base members as well (and for built-in COM on downlevel platforms, it's necessary for a correct vtable layout).

Reply:
Fixed.